core/src/services/routing.js

```diff
@@ -123,20 +123,30 @@ const escapeRegExp = string => {
   return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
 };
 
-const replaceVars = (viewUrl, params, prefix) => {
+const replaceVars = (viewUrl, params, prefix, noParenthesis) => {
```

Comment:
Shall we use a variable called 'parenthesis' so that below we use `parenthesis` instead of `!noParenthesis`? The only changes needed (apart from the usage of the variable) are:
`const replaceVars = (viewUrl, params, prefix, parenthesis = true) => { // line 126`
`viewUrl = replaceVars(viewUrl, componentData.pathParams, ':', false // line 158);`